examples/hello/dune

```diff
@@ -1,3 +1,4 @@
 (executable
   (name main)
-  (libraries eio_main))
+  (libraries eio_main)
+  (modes byte))
```

Comment:
I guess this was for testing something, but we shouldn't complicate the hello-world example.